TheAurorian/src/main/resources/assets/theaurorian/patchouli_books/the_aurorian_guide/zh_cn/entries/modprogression/7moontemplemobs.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "月宫中的生物",
+    "icon": "theaurorian:bossspawnerkeeper",
+    "category": "2modprogression",
+    "sortnum": "7",
+    "readbydefault": true,
+    "pages": [
+        {
+            "type": "entity",
+            "entity": "theaurorian:moonacolyte",
+            "text": "$(#008cff)皎月侍从$()会击退你并在攻击到你时让你漂浮起来。"
+        },
+        {
+            "type": "entity",
+            "entity": "theaurorian:crystallinesprite",
+            "text": "$(#008cff)晶魂$()会在空中向玩家发射火焰。"
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
            "text": "$(#008cff)晶魂$()。"
```
删掉的这一句需要实测。